Makefile

```diff
@@ -155,7 +157,7 @@ copy_node_js:
 ifeq ($(IN_DOCKER),)
 	$(warning Command is designed to be run in the container)
 endif
-	for dest in $(NODE_LIBS_JS) ; do cp $(NODE_MODULES)$$dest $(STATIC_JS) ; done
+	for dest in $(NODE_LIBS_JS) ; do cp -r $(NODE_MODULES)$$dest $(STATIC_JS) ; done
```

Comment:
I suppose an alternative would be to use the lists in settings to do the copying but that's a bit of a yak-shaving excercise.